src/components/Stage.js

```diff
@@ -201,7 +202,7 @@ class Stage extends Component {
             style={{
               padding: "0.5rem",
               position: "absolute",
-              top: "calc(13px - (1.07142857rem * 1.28571429 - 13px) / 2)",

```

Comment:
The reason for having such "precise"  numbers here was to ensure that the loading animation fits pixel-perfectly onto the loaded version. Maybe instead of rounding up, the global rem value could be set inside the style sheet to a nice power of two which would make these values round and known in advance.